src/compass/utils/elevation_antenna_pattern.py

```diff
@@ -0,0 +1,69 @@
+''' A routine to apply elevation antenna pattern correction
+(EAP correction)'''
+
+
+import numpy as np
+from osgeo import gdal
+
+
+def apply_eap_correction(burst, path_slc_vrt, path_slc_corrected, check_eap):
+    '''
+    Apply Elevation Antenna Pattern correction (EAP correction)
+    on the input burst
+
+    Parameters:
+    -----------
+    burst: Sentinel1BurstSlc
+        Input burst
+    path_slc_vrt: str
+        Path to the burst SLC to be corrected
+    path_slc_corrected: str:
+        Path to the burst SLC after EAP correction
+    check_eap: Namespace
+        A namespace that contains flags if phase and/or magnitute
+        EAP correction are necessary
+
+    Return:
+    -------
+    path_burst_corrected:
+        Path to the GDAL-compatible raster after the correction
+
+    '''
+
+    #TODO Indicate into the metadata to indicate if EAP correction is applied by OEPRA SAS
```

Comment:
When the PR gets merged (or also now), submit an issue about adding the application of EAP to the CSLC metadata. So we do not lose track on it

Reply:
Forgot to remove this TODO. The metadata writeout is implemented in `geo_metadata.py`.
I will remove this line.